force-app/main/default/objects/HOT_Request__c/fields/isFileConsent__c.field-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>isFileConsent__c</fullName>
+    <defaultValue>false</defaultValue>
+    <externalId>false</externalId>
+    <label>isFileConsent</label>
```

Comment:
Label er ikke så farlig (men må gjerne ha mellomrom).

Legg til oversettelse for feltet. Det kan hende at det kommer til å bli brukt i rapporter. Fint å bare ha det på plass fra start.